ee/desktop/menu/menu_template.go

```diff
@@ -32,7 +40,26 @@ func (tp *templateParser) Parse(text string) (string, error) {
 		return "", fmt.Errorf("templateData is nil")
 	}
 
-	t, err := template.New("menu_template").Parse(text)
+	t, err := template.New("menu_template").Funcs(template.FuncMap{
+		// hasCapability enables interoperability between different versions of launcher
+		funcHasCapability: func(capability string) bool {
+			if capability == funcRelativeTime {
+				return true
+			}
+			return false
+		},
+		// relativeTime takes a RFC 339 formatted timestamp and returns a fuzzy timestamp
+		funcRelativeTime: func(timestamp string) string {
+			t, err := time.Parse(
+				time.RFC3339,
```

Comment:
When we did the notifications, we ultimately concluded it was simpler to pass seconds-since-epoch, than try to get ruby and go talking via rfc3339. (It shouldn't have been, but 🤷 we took the easy path)

Recommend unix time here as well